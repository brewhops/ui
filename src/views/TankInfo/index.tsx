import React from 'react';

export const TankInfo: React.FC = () => {
  return <div>Tank Info</div>
}
  