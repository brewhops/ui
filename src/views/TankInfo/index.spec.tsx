import React from 'react';
import { TankInfo } from '../TankInfo';
import { shallow } from 'enzyme';

describe('src/pages/TankInfo/index.tsx', () => {
  it('renders the Tank Info page', () => {
    expect(shallow(<TankInfo/>)).toMatchSnapshot();
  })
});