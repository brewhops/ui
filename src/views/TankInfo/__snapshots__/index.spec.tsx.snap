// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/TankInfo/index.tsx renders the Tank Info page 1`] = `ShallowWrapper {}`;
